gh-108901: Deprecate `inspect.getcallargs` and slate it for removal (#112236)
python/cpython: Misc/NEWS.d/next/Library/2023-11-18-17-01-45.gh-issue-108901.T3p5ie.rst

Add a runtime ``DeprecationWarning`` to :func:`inspect.getcallargs` and

> slate it for remove in 3.15.  It was softly-deprecated in 3.5.  Use

Reviewer: ```suggestion
slate it for removal in 3.15.  It was softly-deprecated in 3.5.  Use
```

> slate it for remove in 3.15, since it was softly-deprecated since 3.5, use

Reviewer 2: ```suggestion
slate it for remove in 3.15.  It was softly-deprecated in 3.5.  Use
```

> slate it for removal in 3.15.  It was softly-deprecated in 3.5.  Use
> :meth:`inspect.Signature.bind` instead.

Reviewer 3: ```suggestion
slate it for removal in Python 3.15. It was soft-deprecated in Python 3.5.
Use :meth:`inspect.Signature.bind` instead.
```
